src/app_service/service/community_tokens/service.nim

```diff
@@ -73,10 +75,18 @@ QtObject:
       let data = CommunityTokenDeployedStatusArgs(communityId: tokenDto.communityId, contractAddress: tokenDto.address, deployState: deployState)
       self.events.emit(SIGNAL_COMMUNITY_TOKEN_DEPLOY_STATUS, data)
 
-  proc mintCollectibles*(self: Service, communityId: string, addressFrom: string, password: string, deploymentParams: DeploymentParameters) =
+  proc mintCollectibles*(self: Service, communityId: string, addressFrom: string, password: string, deploymentParams: DeploymentParameters, tokenMetadata: CommunityTokensMetadataDto) =
     try:
       let chainId = self.networkService.getNetworkForCollectibles().chainId
-      let txData = TransactionDataDto(source: parseAddress(addressFrom))
+
+      # TODO this will come from SendModal
+      let suggestedFees = self.transactionService.suggestedFees(chainId)
+      let contractGasUnits = "3702411"
+
+      let txData = ens_utils.buildTransaction(parseAddress(addressFrom), 0.u256, contractGasUnits,
+      if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
+      if suggestedFees.eip1559Enabled: $suggestedFees.maxPriorityFeePerGas else: "",
+      if suggestedFees.eip1559Enabled: $suggestedFees.maxFeePerGasM else: "")
```

Comment:
Could you add an indent on those 3 lines so it's clear that they belong to the txData line.